app/views/lab_results/edit.html.erb

```diff
@@ -0,0 +1,7 @@
+<section class="hero is-small is-info">
+  <div class="hero-body">
+    <p class="title">Edit Lab Result</p>
```

Comment:
Also include a back button on the edit page.

Reply:
resolved